src/routes/auth.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const passportGoogle = require('../utils/google');
+const authControllers = require('../controllers/auth');
+
+const router = express.Router();
+
+router.get(
+  '/signin',
+  passportGoogle.authenticate('google', {
+    scope: ['profile', 'email', 'openid'],
+  })
+);
```

Comment:
Since you will also have password-based login, its probably better to rename this route to something else like `/google`. So that `/signin` route is kept for password-based login.